src/message/messageActionSheet.js

```diff
@@ -301,8 +303,10 @@ export const constructMessageActionButtons = ({
   }
   if (
     message.sender_id === ownUser.user_id
-    // Our "edit message" UI only works in certain kinds of narrows.
-    && (isStreamOrTopicNarrow(narrow) || isPmNarrow(narrow))
+    && (isStreamOrTopicNarrow(narrow)
+      || isPmNarrow(narrow)
+      || isSpecialNarrow(narrow)
+      || isHomeNarrow(narrow))
```

Comment:
Have you tested that editing works in narrows where `isSpecialNarrow(narrow)` or `isHomeNarrow(narrow)` is true?

I wouldn't expect it to work for either of those, because I think we currently refuse to show the compose box in those cases. That's something I hope we can change soon, but it'll take more work toward refactoring `ComposeBox`.

Reply:
Thanks @chrisbobbe for pointing that out.

I just saw the edit message option pop up and assumed it was done, my bad.

Maybe I could help out with the compose box (if necessary) after that PR is merged. Or are there any steps you could suggest to make this work for the meantime?